### ISSUE TITLE
Investigate options for moving to paid tier of Slack for https://instruct-lab.slack.com/

### ISSUE THREAD

**Issue by lhawthorn** [instructlab/community#123]
Problem statement:

We wish to use Slack for conversation amongst InstructLab community members.

We wish to preserve history beyond what's provided with the free Tier of Slack.

Business pricing comes in at 150 USD per user paid annually (12.50 / mo for seat license).

That cost starts getting pretty pricey pretty fast, so we need to investigate what Enterprise pricing options could exist, especially since both the employers of the founding maintainer teams, Red Hat and IBM, are both Enterprise Slack customers.

@lhawthorn will start the process to investigate what's possible from an Enterprise tier pricing perspective. It may be that Slack has some kind of deal for open source project work.

Will speak with the Red Hat OSPO Community Infrastructure team to get this started.

**Comment by lhawthorn** [comment]
So, I will try the process you suggested @jjasghar but word on the street is it's hard to get in that way, especially since InstructLab is not a non-profit nor is it homed in a non-profit foundation. (Though we want to move in that direction sometime, but even then we might be housed in a 501c6 industry trade group vs a 501c3 non-profit.)

I know that it is possible to bridge Slack to Matrix and use this as a means to preserve history past 90 days. Let me see if I can get some help with that route.

**Comment by jjasghar** [comment]
I mean, nothing stopping us from pivoting to matrix now, especially now that our... "restrictions" are gone.

**Comment by lhawthorn** [comment]
Bridging Slack to Matrix is against Slack's ToS. I shall go into a corner and mrph about this for awhile.

**Comment by lhawthorn** [comment]
Decision taken: paid Slack is not currently in scope, we will use mailing lists and GitHub issues for durable conversations

tl;dr: if it's information that needs to stick around a long time or a discussion about something that will set project direction, it goes on the mailing lists (forthcoming real soon now) or GitHub issues; if it's ephemeral, it's great to discuss in Slack. 

In more detail:
As folks have noted, we are using the free tier of Slack which means no preservation of messaging history after 30 days. This is a huge bummer if you want to make sure that project decisions and important information are preserved. Paid Slack is really expensive for an open source project that we aspire to grow to a community of many participants. So for conversations like "please review my PR" or "I need help with this specific error message", or "I just had a hallway chat at a conference with so-and-so and we're going to work on a particular and I'll follow up with an open issue," then Slack is the place. For information we need to make sure is preserved, make sure to use the mailing lists and GitHub issues extensively.

This is not something anyone needs to think too hard about, but please bear in mind you may be gently nudged at times to take a particular discussion in the project Slack workspace to GitHub issues or to the project mailing lists once they are set up.
